cocos/renderer/backend/ProgramCache.h

```diff
@@ -56,6 +56,9 @@ class ProgramCache : public Ref
      */
     backend::Program* newProgram(const std::string& vertexShader, const std::string& fragmentShader);
     
+    /// get built-in program
+    backend::Program* newProgramByProgramType(ProgramType type);
```

Comment:
newProgramByProgramType -> newBuiltinProgram ?

Reply:
Got it